TeamCode/src/main/java/org/firstinspires/ftc/teamcode/movement/MoveRobotLinear.java

```diff
@@ -0,0 +1,72 @@
+package org.firstinspires.ftc.teamcode.movement;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+
+public class MoveRobotLinear {
+    private MovementMotors motorsController;
+    private Power movementPower;
+    private LinearOpMode opMode;
+
+    public MoveRobotLinear(Telemetry telemetry, HardwareMap hardwareMap,
+                           LinearOpMode opMode) {
+        motorsController = new MovementMotors(hardwareMap, telemetry);
+        movementPower = new Power();
+        this.opMode = opMode;
+    }
+
+    private void move(Power direction) {
+        motorsController.setPower(movementPower.multiply(direction));
+    }
+
+    public void moveForward(double speed) {
+        movementPower.setPower(speed);
+        move(Signs.FORWARD);
+    }
+
+    public void moveBackward(double speed) {
+        movementPower.setPower(speed);
+        move(Signs.BACKWARD);
+    }
+
+    public void moveLeft(double speed) {
+        movementPower.setPower(speed);
+        move(Signs.LEFT);
+    }
+
+    public void moveRight(double speed) {
+        movementPower.setPower(speed);
+        move(Signs.RIGHT);
+    }
+
+    public void moveForwardLeft(double speed) {
```

Comment:
Schimba semnatura functiei in `moveForwardLeft(double speed, double angularSpeed)` si dupa faci `movementPower.setPower(new Power(speed, angularSpeed, speed, angularSpeed))` si similar la metoda de mai jos.